pkg/storaged/client.js

```diff
@@ -602,6 +804,31 @@
     for (path in client.jobs) {
         enter_job(client.jobs[path]);
     }
+
+    // UDisks API does not provide a btrfs volume abstraction so we keep track of
+    // volume's by uuid in an object. uuid => [org.freedesktop.UDisks2.Filesystem.BTRFS]
+    // https://github.com/storaged-project/udisks/issues/1232
+    const old_uuids = client.uuids_btrfs_volume;
+    let need_poll = false;
+    client.uuids_btrfs_volume = { };
+    client.uuids_btrfs_blocks = { };
+    for (const p in client.blocks_fsys_btrfs) {
+        const bfs = client.blocks_fsys_btrfs[p];
+        const uuid = bfs.data.uuid;
+        const block_fsys = client.blocks_fsys[p];
+        if ((block_fsys && block_fsys.MountPoints.length > 0) || !client.uuids_btrfs_volume[uuid]) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231214-115043-fd396f8d-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)